cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -36,11 +36,12 @@ def __init__(self, output_file):
         self.maintainers = ['VH']
         self.strict_check = False
         self.use_multithreading = False
-        self.extra_resources = {
-            'switches': {
-                'num_switches': 1
+        if self.current_system.name == 'daint':
+            self.extra_resources = {
+                'switches': {
+                    'num_switches': 1
+                }
```

Comment:
Why don't you just undefine the switches in Dom's configuration? You don't need to change the test then.